index.html

```diff
@@ -49,162 +212,9 @@ <h2 class="job">I'm a software developer</h2>
-            alt="Img_Plaholder"
-          />
-          <div class="project_info">
-            <h4 class="project_name">Multi-Post Stories Gain+Glory</h4>
-            <ul class="languages_used">
-              <li><a href="#">Ruby on rails</a></li>
-              <li><a href="#">css</a></li>
-              <li><a href="#">JavaScript</a></li>
-              <li><a href="#">html</a></li>
-            </ul>
-            <a class="see_project" href="#">See Project</a>
-          </div>
-        </div>
-        <div class="card">
-          <img
-            class="img_placeholder"
-            src="./images/Im_Plaholder2.png"
-            alt="Img_Plaholder"
-          />
-          <div class="project_info">
-            <h4 class="project_name">Multi-Post Stories Gain+Glory</h4>
-            <ul class="languages_used">
-              <li><a href="#">Ruby on rails</a></li>
-              <li><a href="#">css</a></li>
-              <li><a href="#">JavaScript</a></li>
-              <li><a href="#">html</a></li>
-            </ul>
-            <a class="see_project" href="#">See Project</a>
-          </div>
-        </div>
-      </div>
-    </section>
-    <section class="about_me">
-      <div class="detailed_info">
-        <h2 class="title">About me</h2>
-        <p class="aboutme">
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hestiate to contact me.
-        </p>
-        <a class="see_project" href="#">Get My Resume</a>
-      </div>
-      <ul class="card2">
-        <li class="card2_content">
-          <img src="./images/icon-languages.svg" alt="icon-languages" />
-          <h4>Languages</h4>
-          <ul class="languages">
-            <li><a href="#">JavaScript</a></li>
-            <li><a href="#">Ruby</a></li>
-            <li><a href="#">Html</a></li>
-            <li><a href="#">CSS</a></li>
-          </ul>
-        </li>
-        <li class="card2_content">
-          <img src="./images/framework__icon.svg" alt="framework__icon" />
-          <h4>Frameworks</h4>
-          <ul class="languages">
-            <li><a href="#">Bootstrap</a></li>
-            <li><a href="#">Ruby on rails</a></li>
-            <li><a href="#">RSpec</a></li>
-            <li><a href="#">Capybara</a></li>
-            <li><a href="#">Selenium</a></li>
-          </ul>
-        </li>
-        <li class="card2_content">
-          <img src="./images/skills_icon.svg" alt="skills_icon" />
-          <h4>Skills</h4>
-          <ul class="languages">
-            <li><a href="#">Codekit</a></li>
-            <li><a href="#">GitHub</a></li>
-            <li><a href="#">Codepen</a></li>
-            <li><a href="#">Gitlab</a></li>
-            <li><a href="#">Terminal</a></li>
-          </ul>
-        </li>
-      </ul>
-    </section>
-  </body>
-</html>
+      <p>felemenoun@gmail.com</p>
```

Comment:
- [ ] [OPTIONAL]: To keep your project even more professional, you should consider making this a clickable link, that will redirect the user to a new, blank email, addressed to the provided email, you can achieve that by using `<a href="mailto:ifelemenoun@gmail.com">felemenoun@gmail.com</a>`. You can read more about it [here](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#e-mail_links).